File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Shippings/IShippingData .php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Shippings;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IShippingData extends IShippingAddress,IShippingEmail,IShippingFax,IShippingPhone,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Shippings/IShippingData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Shippings
{
    public interface IShippingData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IShippingAddress, IShippingPhone, IShippingEmail, IShippingFax

    {
        
    }
}
